vsi2tif/vsi2tif.py

```diff
@@ -17,6 +17,7 @@ def main():
     parser.add_argument("-q", "--quality", help="compression quality used with JPEG compression", default=85)
     parser.add_argument("-m", "--max-mem", help="set maximum memory in the java vm", default=32)
     parser.add_argument("-v", "--verbose", help="set verbosity level", default=1, type=int)
+    parser.add_argument("-rs", "--remove-name-spaces", help="replace spaces in filename with underscores in batch mode", default=False)
```

Comment:
@dbouget probably makes more sense to make this a flag, using `action='store_true'`?
https://docs.python.org/3/library/argparse.html#core-functionality

That way this is always false and only enabled if you append `--remove-name-spaces` flag to the run cmd.

Reply:
@dbouget Can you address this?

Reply:
This was done in the last commit